scalahost/nsc/src/main/scala/scala/meta/internal/semantic/AttributesOps.scala

```diff
@@ -147,7 +147,7 @@ trait AttributesOps { self: DatabaseOps =>
                 if (symbol == m.Symbol.None) return
 
                 names(mtree.pos) = symbol
-                if (mtree.isBinder) {
+                if (config.semanticdb.isFat || mtree.isBinder) {
```

Comment:
Why restrict this just to fat semantic databases? I can imagine slim databases also wanting this functionality.